data/src/config.rs

```diff
@@ -8,20 +8,24 @@ use thiserror::Error;
 pub use self::buffer::Buffer;
 pub use self::channel::Channel;
 pub use self::dashboard::Dashboard;
+pub use self::server::Server;
+
+use crate::environment;
 use crate::palette::Palette;
-use crate::{environment, server};
+use crate::server::Map;
```

Comment:
Map here has no context, I'd keep the module namespace for clarity... `server::Map` vs `Map`